client/extensions/woocommerce/state/data-layer/orders/index.js

```diff
@@ -4,30 +4,63 @@
  */
 import debugFactory from 'debug';
 import { isFinite, omitBy } from 'lodash';
+import { translate } from 'i18n-calypso';
 import qs from 'querystring';
 
 /**
  * Internal dependencies
  */
-import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { dispatchRequest, dispatchRequestEx } from 'state/data-layer/wpcom-http/utils';
```

Comment:
Still groking data layer, but `dispatchRequestEx` allows you to do a `.del` here?

Reply:
No, it's a new version of the `dispatchRequest` function, updated based on general use. One change is that you can return an action directly, rather than the dispatch function; see the difference between [`del`](https://github.com/Automattic/wp-calypso/blob/314c5011baa24d987d1b093fb1c23739581b81d1/client/extensions/woocommerce/state/data-layer/orders/index.js#L37:L40) & [`requestOrders`](https://github.com/Automattic/wp-calypso/blob/314c5011baa24d987d1b093fb1c23739581b81d1/client/extensions/woocommerce/state/data-layer/orders/index.js#L79:L85). It also merged all the arguments into one object, rather than positional args. See p4TIVU-8LU-p2